### PR TITLE
Resource managers see holidays as background events in resource calendar

### PR DESCRIPTION
**Pull request by AndrewVos**

### CODE REVIEW

**Comment by benlovell** [comment]
Can we get some coverage of this in a feature spec? Just to be sure that the calendar bits all hook up and don't regress.

File: app/controllers/holidays_controller.rb [hunk 1]
```diff
@@ -1,11 +1,17 @@
 class HolidaysController < ApplicationController
   before_action :authorise_for_resource_managers!
 
+  def merged
```
Comment (benlovell) [inline app/controllers/holidays_controller.rb]: Am I missing something or is this not actually being used?

Comment (benlovell) [inline app/controllers/holidays_controller.rb]: Wait, ignore that.

Comment (AndrewVos) [inline app/controllers/holidays_controller.rb]: It's being used for `holidays#index`

**Pull request merged**